CHANGELOG.md

```diff
@@ -1,19 +1,26 @@
 # laminr v1.0.0
 
-This is a major rewrite of the package, resulting in many breaking changes but also offering an interface that is much more consistent with the Python API. Please see the updated ["Get started"](https://laminr.lamin.ai/articles/laminr.html) vignette for more information.
+This is a major rewrite of the package to offer an interface that has feature-parity with the Python API (PR #146).
 
-## BREAKING CHANGES
+Please see the updated ["Get started"](https://laminr.lamin.ai/articles/laminr.html) vignette for more information.
+
+Migration guide:
+
+- Replace `db <- connect()` with `ln <- import_module("lamindb")`. The `ln` object behaves in the same way as the `lamindb` module object in Python and is largely similar to the `db` object in **{laminr}** < v1.
```

Comment:
```suggestion
- Replace `db <- connect()` with `ln <- import_module("lamindb")`. The `ln` object behaves in the same way as the `lamindb` module object in Python and fulfills the same role as the `db` object in **{laminr}** < v1.
- The interface now matches the Python interface. Please refer to the [`lamindb` documentation](https://docs.lamin.ai/) for more information on syntax.
```

Reply:
Can we say that `db` and `ln` have essentially the same calls rather than pointing the user to the Python documentation?

Is there any case where `db` and `ln` are **not** the same?

Reply:
🙏